static/js/subs.js

```diff
@@ -515,6 +527,19 @@ exports.setup_page = function (callback) {
     // continue the strategy that we re-render everything from scratch.
     // Also, we'll always go back to the "Subscribed" tab.
     function initialize_components() {
+        // Sort by name by default when opening "Manage streams".
+        sort_order = "by_stream_name";
```

Comment:
This seems to use a different `_` vs. `-` decision than `key`; is that intentional?

Reply:
It was intentional (to mimic what seemed to be the style of the surrounding code - it looked like stuff in the DOM uses hyphens while Zulip JS usually uses underscores) but wasn't a rigorous design decision or anything. Should I just switch this to hyphens?

Reply:
I ended up moving some things around and just directly using the hyphenated version.